go.mod

```diff
@@ -123,7 +123,7 @@ require (
 	k8s.io/klog/v2 v2.9.0
 	k8s.io/kube-aggregator v0.0.0
 	k8s.io/kube-controller-manager v0.0.0
-	k8s.io/kube-openapi v0.0.0-20210421082810-95288971da7e
+	k8s.io/kube-openapi v0.0.0-20211109043538-20434351676c
```

Comment:
@dims we require tags, don't we?

Reply:
hmm... we don't have any tags in k8s.io/kube-openapi currently, and as soon as we add one on any branch, it will be the semantically largest one

I'm not eager to figure out a tagging strategy for that repo in the timeframe these PRs want

Reply:
+1 to what @liggitt said. let's please do try in the future, not just now.